lib/bolt_spec/plans.rb

```diff
@@ -164,6 +166,11 @@ def inventory
       @inventory ||= Bolt::Inventory.new({})
     end
 
+    # Override in your tests
```

Comment:
I don't think we should expose this to test authors. I think we should expose a way to make assertions about what `out::message` calls occur in a plan.

One option for that would be to implement our own event publisher and write allow/expect methods for it.
Another option would be to add an executor method that `out::message` calls to the executor and use that for testing.